eva-pipeline/src/main/java/embl/ebi/variation/eva/pipeline/jobs/VariantAnnotConfiguration.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2015 EMBL - European Bioinformatics Institute
```

Comment:
Only for the files that have been created/modified in 2016. So it could not apply exactly to the file I have highlighted, but to others.

If there were created in 2015, then the copyright declaration should read "2015-2016", and if they were created this year, just "2016".